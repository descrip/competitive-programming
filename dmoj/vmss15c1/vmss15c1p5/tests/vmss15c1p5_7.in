8 899 965
asdfghjkl 540 8 27
candy 563 24 2
dynamicprogramming 912 13 31
email 777 15 19
quack 149 12 6
stromae 410 8 16
www 239 7 31
zxcvb 426 1 21
