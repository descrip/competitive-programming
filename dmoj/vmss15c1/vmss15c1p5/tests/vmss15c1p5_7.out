42228
asdfghjkl 0
candy 6
dynamicprogramming 0
email 50
quack 0
stromae 0
www 0
zxcvb 0
